jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as WebSocket from 'ws';
+import WebSocketServer = WebSocket.Server;
+import * as rpc from 'noice-json-rpc';
+import Crdp from 'chrome-remote-debug-protocol';
+
+export interface ChromeDevToolsProxyServerOptions extends WebSocket.ServerOptions {
+}
+
+export class ChromeDevToolsProxyServer {
+  constructor(options: ChromeDevToolsProxyServerOptions) {
+    const wss = new WebSocketServer(options);
+    const rpcServer = new rpc.Server(wss);
+    const api: Crdp.CrdpServer = rpcServer.api();
+
```

Comment:
@grgustaf this is currently not working at all.

I found the `noice-json-rpc` and `chrome-remote-debug-protocol` projects (both by the same author) while searching for existing libraries we could re-use. It looks very useful but I didn't get the example code to work out of the box just copy & pasting it. I didn't spend much time on it so I probably missed something silly.